src/main/docs/guide/hibernate/hibernateJoinQueries.adoc

```diff
@@ -19,7 +19,7 @@ snippet::example.ProductRepository[project-base="doc-examples/hibernate-example"
 
 <1> The ann:data.annotation.Join[] is used to indicate a `JOIN FETCH` clause should be included.
 
-Note that the ann:data.annotation.Join[] annotation is repeatable and hence can be specified multiple times for different associations. In addition, the `type` member of the annotation can be used to specify the join type, for example `LEFT`, `INNER` or `RIGHT`.
+Note that the ann:data.annotation.Join[] annotation is repeatable and hence can be specified multiple times for different associations. In addition, the `type` member of the annotation can be used to specify the join type, for example `LEFT`, `INNER`, `RIGHT` or `OUTER`.
```

Comment:
Should this be added to hibernate docs? Don't see outer supported here in [JpaQueryBuilder](https://github.com/micronaut-projects/micronaut-data/blob/master/data-model/src/main/java/io/micronaut/data/model/query/builder/jpa/JpaQueryBuilder.java#L200)

Reply:
A mistake, I will remove it.